db/data/seed_tag_states.sql

```diff
@@ -1,5 +1,5 @@
 -- Set up the states
-INSERT INTO TagState VALUES 
+INSERT INTO Tag_State VALUES 
```

Comment:
Are there any corresponding db_access changes? Did a quick look at it doesn't look like we're currently doing any query/joins to this table, just double checking though.